app/environments/brassbirmingham/brassbirmingham/envs/classes/road_location.py

```diff
@@ -0,0 +1,59 @@
+from __future__ import annotations
+
+from typing import TYPE_CHECKING, List, Optional
+
+from python.id import id
+
+if TYPE_CHECKING:
+    from .roads.road import Road
+    from .town import Town
+    from .trade_post import TradePost
+
+
+class RoadLocation:
+    """
+    Road Location - Where a canal or railroad may be built
+
+    :param networks: Networks to what Towns/TradePosts road is networked to, array of Towns/TradePosts
+    :param canBuildCanal=True: is river
+    :param canBuildRailroad=True: is railroad track
+    """
+
+    def __init__(
+        self,
+        networks: List[Town | TradePost],
```

Comment:
A particular road can only be connected to 2 towns or tradeposts, right? Maybe we should make this something more specifically suited for 2 elements instead of a generic List to prevent confusion?

Reply:
Geez, they just had to be special 🤦